### PR TITLE
Discovery was missed during the initial ansible work

### PR DESCRIPTION
**Pull request by jibsheet**
This let a machine boot without migrations running.

@edx/devops
@clintonb

### CODE REVIEW

File: playbooks/roles/supervisor/files/pre_supervisor_checks.py [hunk 1]
```diff
@@ -111,6 +112,15 @@ def edp_for_instance(instance_id):
...
         help="Location to of the credentials code.")
 
+    discovery_migration_args = parser.add_argument_group("discovery_migrations",
+            "Args for running discovery migration checks.")
+    discovery_migration_args.add_argument("--discovery-python",
+        help="Path to python to use for executing migration check.")
+    discovery_migration_args.add_argument("--discovery-env",
+        help="Location of the discovery environment file.")
+    discovery_migration_args.add_argument("--discovery-code-dir",
+        help="Location to of the discovery code.")
```
Comment (clintonb) [inline playbooks/roles/supervisor/files/pre_supervisor_checks.py]: Remove "to".

**Comment by clintonb** [comment]
:+1:

**Comment by fredsmith** [comment]
:+1:

**Comment by feanil** [comment]
:+1:

**Pull request merged**